int64 imgID
sensor_msgs/RegionOfInterest roi
---
Metrics metrics